vault/resource_generic_secret.go

```diff
@@ -47,12 +47,6 @@ func genericSecretResource() *schema.Resource {
 				Sensitive:    true,
 			},
 
-			"allow_read": {
```

Comment:
Not sure why this is deleted?

Reply:
It was previously marked as `Removed`, but was not  properly removed in https://github.com/hashicorp/terraform-provider-vault/pull/1175/files#diff-bcbec1f006c1e5fef492a7d3fb86cdc7c589b4cc0166230a8ed27b933030fdddL54